virttest/storage.py

```diff
@@ -18,6 +18,7 @@
 from avocado.core import exceptions
 from avocado.utils import process
 
+from virttest import ssh
```

Comment:
Suggest the name `storage_ssh` for this module to avoid name conflicts.

Reply:
changed it to libssh